platforms/hyperledger-indy/charts/indy-ledger-txn-new-node/Chart.yaml

```diff
@@ -0,0 +1,5 @@
+apiVersion: v1
+appVersion: "2.0"
+description: A Helm3 chart for Running Indy Ledger Script for Issuing a NYM Transaction
+name: indy-ledger-txn-new-node
```

Comment:
Why do we need a new chart for running ledger-txn for a new node? Why can't we use the existing charts?